test/e2e/test-cmd.sh

```diff
@@ -59,7 +121,35 @@ EOF
 
 kubectl get controllerrevisions sample --namespace=default
 
-# TODO: run the migrator
+# STEP 2: change the apiserver configuration of --storage-versions.
+
+# Change the apiserver manifest to change the storage version.
+gcloud compute --project "${PROJECT}" ssh --zone "${KUBE_GCE_ZONE}" "${CLUSTER_NAME}-master" --command \
+"sudo sed -i \"s/--v=/--storage-versions=apps\/v1beta2 --v=/\" /etc/kubernetes/manifests/kube-apiserver.manifest"
```

Comment:
so the k8s needs to support apps/v1beta2, did you fix the k8s release in this test?

Reply:
Good point, we'll need to update the config [here](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml#L30) to pin the k8s version. On the other hand, I want to test the migrator with the latest Kubernetes to ensure compatibility. In the future, we can check the apiserver version in this test script, or switch to a more dynamic test setup. I don't know which path we'll take yet, so I left a TODO.